### ISSUE TITLE
Dynamic key_suffix issue.

### ISSUE THREAD

**Issue by orehmane** [ixti/sidekiq-throttled#22]
As per the docs, I wrote the following proof of concept worker for using dynamic key suffixes:

```
class PocWorker
  include Sidekiq::Worker
  include Sidekiq::Throttled::Worker

  sidekiq_throttle({ concurrency: { limit: 1, key_suffix: -> (id) { id } } })

  def perform(id)
    Sidekiq.logger.info("Starting #{id}...")
    thing = Thing.find(id)
    OtherThing.create(name: Time.now.to_s, thing: thing)
    (1..99999).each { |n| STDERR.puts n }
    Sidekiq.logger.info("Ending #{id}...")
  end
end
```

The idea is that I would call `PocWorker.perform_async(1)` 4 times, and then call `PocWorker.perform_async(2)` 4 times, and thought I would see `OtherThing`s created 2 at a time, but they were created 1 at a time instead.  I think I'm following the docs correctly, is there an issue with my code?

Using `sidekiq-throttled 0.6.1` and `sidekiq 4.1.3`.  This happened both locally and on a staging environment with multiple workers available.

Thank you for your help!

**Comment by ixti** [comment]
There was a regression introduced in `0.6.0` and fixed in `0.6.5`, which broke concurrency throttling when used with `redis-namespace`. Can you please test against `0.6.5` and tell if it still an issue for you or not?

**Comment by orehmane** [comment]
It still seems to be happening.

**Comment by ixti** [comment]
Will try to check it today.

**Comment by iporsut** [comment]
I found point that mistake in this https://github.com/sensortower/sidekiq-throttled/blob/master/lib/sidekiq/throttled/strategy.rb#L32

32: `concurrency[:key_suffix] = key_suffix`

38: `threshold[:key_suffix] = key_suffix`

You replace :key_suffix key in hash concurrency and threshold with nil

**Comment by ixti** [comment]
@iporsut Oh! Good catch! Will fix ASAP, or feel free to open a PR

**Comment by ixti** [comment]
Fix released as `v0.6.6`, kudos to @iporsut;
